### PR TITLE
Add debian and ubuntu14 to the runtime list for packages cache

### PR DESCRIPTION
**Pull request by troydai**

### CODE REVIEW

**Comment by troydai** [comment]
/cc @pranavkm @javiercn @glennc 

Could you please review at your best convenience?

File: src/SplitPackages/ProjectJsonFileBuilder.cs [hunk 1]
```diff
@@ -131,7 +131,7 @@ public void Execute()
 
         private IDictionary<string, JObject> CreateRuntimesDictionary()
         {
-            return new[] { Runtime.Win7x64, Runtime.Win7x86 }.ToDictionary(r => r.Name, r => new JObject());
+            return Runtime.AllRuntimes.ToDictionary(r => r.Name, r => new JObject());
```
Comment (ryanbrandenburg) [inline src/SplitPackages/ProjectJsonFileBuilder.cs]: Where does `AllRuntimes` come from? I don't see it in `src/SplitPackages/Runtime.cs`, is it an extension from somewhere?

Comment (troydai) [inline src/SplitPackages/ProjectJsonFileBuilder.cs]: Isn't it in the Runtime.cs on line 13?

Comment (ryanbrandenburg) [inline src/SplitPackages/ProjectJsonFileBuilder.cs]: Ah yep, missed it.

**Comment by ryanbrandenburg** [comment]
Looks :shipit: to me.

**Comment by troydai** [comment]
Thanks

**Pull request merged**